src/server/enrollment-system.js

```diff
@@ -362,6 +359,10 @@ function veteranToDependentFinancialsCollection(veteran) {
  * @returns {Object} ES system dependentFinancialsCollection message
  */
 function veteranToSpouseFinancials(veteran) {
+  if (!_.includes(['Married', 'Separated'], veteran.maritalStatus)) {
```

Comment:
Looks like if they aren't married, this will cause none of the financial info to be returned, since it looks like on lines 925-926 below that some children financial info is returned. Do we still want to return that regardless of whether or not they have a spouse?